reflex/components/overlay/banner.py

```diff
@@ -79,7 +79,11 @@ def create(cls, comp: Optional[Component] = None) -> Component:
         if not comp:
             comp = Text.create(*default_connection_error())
         return super().create(
-            header="Connection Error",
-            body=comp,
-            is_open=has_connection_error,
+            has_connection_error,
+            Modal.create(
+                header="Connection Error",
+                body=comp,
+                is_open=has_connection_error,
+            ),
+            Fragment.create(),
```

Comment:
Cond doesn't need a second argument

Reply:
Though I noticed the type annotation for `Cond.create` is wrong as we don't set a default value of None - we should fix that.

Reply:
> we should fix that.

on it